x/ccv/types/events.go

```diff
@@ -2,11 +2,38 @@ package types
 
 // CCV events
 const (
-	EventTypeTimeout      = "timeout"
-	EventTypePacket       = "ccv_packet"
-	EventTypeChannelClose = "channel_closed"
+	EventTypeTimeout                  = "timeout"
+	EventTypePacket                   = "ccv_packet"
+	EventTypeChannelClose             = "channel_closed"
+	EventTypeChannelEstablished       = "channel_established"
+	EventTypeFeeTransferChannelOpened = "fee_channel_opened"
+	EventTypeConsumerClientCreated    = "consumer_client_created"
+
+	EventExecuteConsumerChainSlash = "execute_consumer_chain_slash"
+	EventTypeFeeDistribution       = "fee_distribution"
+	EventTypeSendSlashPacket       = "send_slash_packet"
+	EventTypeSendMaturedVSCPacket  = "send_matured_vsc_packet"
```

Comment:
Some of these are in past tense and some are in present tense, could it be more consistent?

By the way there is a nice vscode extension `change-case` which lets you easily change Camel to snake ect. Some of these dont match (ChannelClose = "channel_closed")

Reply:
Changed cases that did not match.